src/inspector/Inspector.zig

```diff
@@ -14,6 +14,7 @@ const input = @import("../input.zig");
 const renderer = @import("../renderer.zig");
 const terminal = @import("../terminal/main.zig");
 const inspector = @import("main.zig");
+const utils = @import("utils.zig");
```

Comment:
Thanks for calling this out. I try to avoid `utils` packages since it becomes a dumping ground for anything and everything. I'd prefer naming this `units.zig`

Reply:
@mitchellh I had the same feeling initially. Cool, I've adjusted based on your feedback. 

Thanks a ton.